policyengine_us/variables/household/demographic/person/pregnancies.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_us.model_api import *
+
+
+class preganant_expected_children(Variable):
+    value_type = int
+    entity = Person
+    definition_period = YEAR
+    label = "The number of children a pregnant person is expecting"
+    adds = ["is_pregnant"]
```

Comment:
Let's instead go the other way: add `adds = ["current_pregnancies"]` in `is_pregnant`.